### PR TITLE
Expand alt text descriptions

### PR DESCRIPTION
**Pull request by tobyhodges**
Partially addresses #429 by expanding alt text descriptions for images in episode 3. I also moved one part of a paragraph below the relevant figure since I think the lesson content makes more sense that way around.

### CODE REVIEW

**Comment by tobyhodges** [comment]
> I just hope I have not crossed your paths in https://github.com/carpentries-lab/deep-learning-intro/pull/596 (I removed the y axis labels there because there no longer reflected what the code generates).

Addressed in latest commit.

File: episodes/3-monitor-the-model.md [hunk 1]
```diff
@@ -807,7 +808,7 @@ history = model.fit(X_train, y_train,
 plot_history(history, ['root_mean_squared_error', 'val_root_mean_squared_error'])
 ```
 
-![](fig/03_training_history_5_rmse_batchnorm.png){alt='Output of plotting sample'}
+![](fig/03_training_history_5_rmse_batchnorm.png){alt='Plot of error vs epochs for the training set and the validation set displaying similar performance across the two sets. RMSE for the validation set drops more than mean squared error for the training set at first, tracks the training error until approximately 50 epochs, then begins to gradually increase while error for the training set continues to gradually decrease.'}
```
Comment (carschno) [inline episodes/3-monitor-the-model.md:811]: > RMSE for the validation set drops more than mean squared error for the training set at first,

It might not be clear that `mean squared error` is actually the same as `RMSE` here.

**Review by carschno — APPROVED** [review]
Looks fine! Just one nitpicky suggestion, feel free to ignore.
